### ISSUE TITLE
Fatal error can't decode packed binary

### ISSUE THREAD

**Issue by haiyan-MS** [Nesvilab/FragPipe#466]
- Describe the issue or question:

FATA[11:00:19] Cannot decode packed binary. EOF   

- Upload your log file
(If a log file hasn't been generated, go to the 'Run' tab in FragPipe, click 'Export Log', zip the resulting "log_[date_time].txt" file to avoid truncation, then attach the zipped file by drag & drop here.
[log_2021-09-22_12-32-55.txt](https://github.com/Nesvilab/FragPipe/files/7212420/log_2021-09-22_12-32-55.txt)
)

**Comment by fcyu** [comment]
Felipe @prvst can you take a look? Everything looks good except for the Philosopher filter command.

Best,

Fengchao

**Comment by prvst** [comment]
I don't see anything wrong. I wonder if the interact files are using commas instead of dots.

**Comment by haiyan-MS** [comment]
Hi Thanks for the fast response. Maybe this is not a software issue but more of my problem? I was wondering if you can point me out to a place where I can get to know more about how to use it. I have looked at the tutorials. But they are quite simple. for example, TMT workflow, I can't find a place to do the TMT isotope correction. Also, there is not much said about validation. If I choose validation on one of the Prophet, it will have error tell me can't find those files.
 If I ignore validation, try an open search, and crystal C as default validation, I only get one tsv file. It has peptide and protein information, but missing for example, protein groups, or peptide position in protein groups or site confidence.

I was wondering if I can get more help somewhere.

Haiyan

________________________________

**Comment by fcyu** [comment]
Normally, we suggest you load the workflow according to your application and change some essential parameters, such as database, variable modifications, enzymes. Most of the parameters work well with their default values. All those tools, include the Prophets, work together to get the result. If you don't know what exactly those tools are doing, you can just use the default settings from workflow.

Here (https://github.com/Nesvilab/FragPipe) has all tutorials. Taking open search (https://fragpipe.nesvilab.org/docs/tutorial_open.html) for example, you only need to load your files and database, specify the result folder, and run.

As to TMT workflow, we don't support TMT isotope correction.

Best,

Fengchao

**Comment by haiyan-MS** [comment]
Hi Fengchao:

Thank you for your clarification.

However, the error message is exactly what I did as you said, my fasta and my run with everything default.

Haiyan

________________________________

**Comment by fcyu** [comment]
Right, I also don't know what caused the error. Need Felipe @prvst to take a look since the error is from Philosopher.

Best,

Fengchao

**Comment by prvst** [comment]
send us the interact.(pep|pro).xml, please

**Comment by haiyan-MS** [comment]
The file is too big to be sent as attachement. here is the box link you can download. Thanks.

https://rutgers.box.com/s/jr1yu4tznou8pgcbjurxzdd9xvjaa895

________________________________

**Comment by haiyan-MS** [comment]
HI Felipe:

I was wondering if you have figured out what did I do wrong?

Also, a question: For phosphorylation, do you have also calculate site confidence? Sometimes it is required for publication.

Thanks for your help.

Haiyan

________________________________

**Comment by prvst** [comment]
not yet, the file seems ok at first look, we might need to run the pipeline on it to test it.

**Comment by fcyu** [comment]
Hi Ben,

This is a known issue (https://github.com/Nesvilab/FragPipe/issues/444). Please change your operating system's local for now.

Best,

Fengchao
